deploy/csi-azuredisk-controller.yaml

```diff
@@ -37,6 +37,13 @@ spec:
           volumeMounts:
             - mountPath: /csi
               name: socket-dir
+          resources:
+            limits:
+              cpu: 2m
```

Comment:
- for limits.cpu, I think it could be `200m`, 2m/20m is really too small.
 - for limits.memory, I think it could be `200m`.

Reply:
I tested with just 2m/20Mi limit and so far no issues. But I agree that 200m/200Mi is safer. Changed to that value.